prov/shm/src/smr_progress.c

```diff
@@ -198,29 +198,24 @@ static int smr_progress_iov(struct smr_cmd *cmd, struct iovec *iov,
 	return -ret;
 }
 
-static int smr_progress_multi_recv(struct smr_ep *ep, struct smr_queue *queue,
-				   struct smr_ep_entry *entry, size_t len)
+static void smr_progress_multi_recv(struct smr_ep *ep, struct smr_queue *queue,
+				    struct smr_ep_entry *entry,
+				    struct smr_ep_entry *dup_entry, size_t len)
```

Comment:
The call takes a 'dup_entry' as input, but all it wants to do with it is return the flags from entry, with one of the bits cleared.  And that's only in some cases.